mpas_analysis/config.default

```diff
@@ -813,6 +813,49 @@ normArgsDifference = {'vmin': -0.5, 'vmax': 0.5}
 # place the ticks automatically by default
 # colorbarTicksDifference = numpy.linspace(-0.5, 0.5, 9)
 
+[climatologyMapBGC]
+## options related to plotting climatology mpas of BGC
+
+# variables to plot
+variables = ['PO4', 'NO3']
+
+# Times for comparison times (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['JFM', 'JAS', 'ANN']
+
+# the first year over which to average climatalogies
+startYear = 11
+# the last year over which to average climatalogies
+endYear = 20
+
+# Times for comparison times (Jan, Feb, Mar, Apr, May, Jun, Jul, Aug, Sep, Oct,
+# Nov, Dec, JFM, AMJ, JAS, OND, ANN)
+seasons =  ['JFM', 'JAS', 'ANN']
+
+# comparison grid(s) ('latlon', 'antarctic') on which to plot analysis
+comparisonGrids = ['latlon']
+
+# colormap for model/observations
+colormapNameResult = BuOr
+# the type of norm used in the colormap
+normTypeResult = linear
+# A dictionary with keywords for the SemiLogNorm
+normArgsResult = {'vmin': 33.8, 'vmax': 35.0}
+# place the ticks automatically by default
+# colorbarTicksResult = numpy.linspace(34.2, 35.2, 9)
```

Comment:
The way things are currently written,  it will be necessary to make a config section for each BGC variable you want to plot, e.g. 
```
[climatologyMapBGCPO4]
# colormap for model/observations
colormapNameResult = BuOr
...
```
so that you can define a different color map and range for each.

It might be worth making a separate `config.bgc` if this ends up being a *lot* and this would get loaded in addition to `config.default` before the custom config file.

If you can think of a cleaner way to do this, I'm happy to help support it, but I don't see an easy way to avoid this.